### ISSUE TITLE
cfn-deployer is unable to redeploy a stack if previous deployed version has been deleted

### ISSUE THREAD

**Issue by boardthatpowder** [alexa/ask-cli#210]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[X ] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Other... Please describe:
</code></pre>

If a stack is deleted that was previously created by the cfn-deployer, any subsequent deployments fail with the following error message (sensitive data redacted):
```
==================== Deploy Skill Infrastructure ====================
  ✖ Deploy Alexa skill infrastructure for region "default"
[Error]: The CloudFormation deploy failed for Alexa region "default": ValidationError: Stack:arn:aws:cloudformation:us-west-2:000000000000:stack/xxxxxxxxxx/ee967d20-a0eb-11ea-b0e0-06a318773a24 is in DELETE_COMPLETE state and can not be updated.
```

## Expected Behavior
cfn-deployer should redeploy as a new stack if it detects the presence of a deleted one.

## Current Behavior
Deployment fails - see summary.

## Steps to Reproduce (for bugs)
- Configure a skill to be deployed via cfn-deployer
- `ask deploy`
- Delete the cloudformation stack
- `ask deploy`

## Possible Solution
Do not attempt an update a stack with DELETE_COMPLETE status.  Instead, create a new one.

A temporary workaround is to delete `profiles.<profileName>.skillInfrastructure.@ask-cli/cfn-deployer.deployState.stackId` from `<project_root>/.ask/ask-states.json`.

## Your Environment and Context
* ask-cli version: 2.9.0
* Operating System and version: macOS Catalina 10.15.4
* Node.js version used for development: v12.13.1 
* NPM version used for development: 6.13.4

**Comment by kakhaUrigashvili** [comment]
@boardthatpowder opened a PR: https://github.com/alexa/ask-cli/pull/224

**Comment by kakhaUrigashvili** [comment]
fixed in version 2.11.0
